Code/src/main/java/nl/utwente/viskell/ui/components/ValueBlock.java

```diff
@@ -22,16 +24,17 @@
  */
 public class ValueBlock extends Block {
     /** The value of this ValueBlock. */
-    private StringProperty value;
+    protected StringProperty value;
```

Comment:
There are public `(get|set)value(property)` methods available. You could just use those from ValueBlock's subclasses.